meta/main.yml

```diff
@@ -7,7 +7,7 @@ galaxy_info:
   description: 'Manage network interface configuration in /etc/network/interfaces'
   company: 'DebOps'
   license: 'GPL-3.0'
-  min_ansible_version: '2.0.0'
+  min_ansible_version: '2.2.0'
```

Comment:
https://github.com/drybjed/ansible-ifupdown/blob/5668d70ac81ea8fa7bbb56da3c60e44efa984dd2/docs/introduction.rst#installation could also be updated. https://github.com/ypid/ypid-ansible-common/blob/master/bin/ansible-update-min-version would do it for you. Refer to the example usage I just added :)